docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,97 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`MicroK8s is installed <set_microk8s>`.
+* :ref:`DSS CLI is installed <install_DSS_CLI>`.
+* The Intel GPU is recognised by your machine. This can be confirmed by running `intel_gpu_top` from the `intel-gpu-tools` package. For discrete GPUs on older versions of Ubuntu like 22.04, you may need to perform additional steps like install a HWE kernel.
```

Comment:
Let's only mention Intel GPU here, and have a dedicated section for verifying the Intel GPU down below with the exact commands someone needs to write to verify the GPU is visible

Reply:
Although, looking at the NVIDIA how-to, we don't have as a requirement to have an NVIDIA GPU there. @afgambin should we include the need of the GPU in the prerequisites of both guides?

Reply:
@kimwnasptd I have a question about the requirements. The second requirement (installation of the DSS CLI) has a link to a tutorial, and in that tutorial there is a section about installing MicroK8s, so MicroK8s is a requirement for DSS. In that case, isn't the first point (MicroK8s) redundant, since users will already have it installed when following the tutorial for setting up DSS.

Reply:
Good points. Regarding Kimonas' suggestion about a dedicated section for verifying the GPU, let's go for it.

Regarding about intalling MicroK8S, I agree it is implicit in the DSS installation. So, yes, I would remove the first item (also, for the NVIDIA guide, can you do it as well Manos as part of this PR)?

So, prerequisites are:
- DSS installed (link to get started tutorial)
- Intel/NVIDIA GPU on your machine.

Reply:
Don't have a hard preference. Indeed only linking to having DSS initialised and point to the tutorial could handle users installing MicroK8s the way we want it.

SGTM to only point to havind DSS initialised

Reply:
I agree!